src/sharedServices/dynamoDB/dynamoDBUpdate.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  UpdateItemCommand,
+  UpdateItemCommandInput
+} from '@aws-sdk/client-dynamodb'
+import { DynamoDBParams } from '../../types/dynamoDBParams'
+import { isDataRequestParams } from '../../types/dataRequestParams'
+import { getEnv } from '../../utils/helpers'
+import { ddbClient } from './dynamoDBClient'
+
+export const updateQueryByZendeskId = async (
+  zendeskId: string,
+  attributeKey: string,
+  attributeValue: string
+): Promise<DynamoDBParams> => {
+  const params: UpdateItemCommandInput = {
+    TableName: getEnv('DYNAMODB_TABLE_NAME'),
+    Key: { zendeskId: { S: zendeskId } },
+    ReturnValues: 'ALL_NEW',
+    ExpressionAttributeValues: { ':value': { S: `${attributeValue}` } },
+    UpdateExpression: `SET ${attributeKey}=:value`
+  }
+
+  const updatedData = await ddbClient.send(new UpdateItemCommand(params))
+  const responseObject = updatedData?.Attributes
+
+  if (!responseObject) {
+    throw new Error(
```

Comment:
Do we need to do anything here? Like close the Zendesk ticket?

Same question for the `throw` further down too!

Reply:
I've added a means to handle errors and update the zendesk ticket in the handler